examples/stub_domains/main.tf

```diff
@@ -19,12 +19,12 @@ locals {
 }
 
 provider "google" {
-  version = "~> 2.2"
+  version = "~> 2.2.0"
```

Comment:
I found you need the `2.3.0` provider to have the module work as is. The set of provider versions that works currently is pretty narrow... for the moment it might be best to move to the `=` exact version spec.

Reply:
Locked to 2.3.x